Dashboard/app/js/lib/controllers/data-controllers.js

```diff
@@ -106,6 +107,10 @@ FLOW.cascadeResourceControl = Ember.ArrayController.create({
 		FLOW.store.findQuery(FLOW.Question, {cascadeResourceId: FLOW.selectedControl.selectedCascadeResource.get('keyId')});
 	}.observes('FLOW.selectedControl.selectedCascadeResource'),
 
+	triggerStatusUpdate: function(){
+		this.set('statusUpdateTrigger',!this.get('statusUpdateTrigger'));
```

Comment:
There is a built-in feature for this: `this.toggleProperty('statusUpdateTrigger')` do you think is worth changing?